dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -756,8 +756,13 @@ private boolean checkTaskHasSubProcess(String taskType) {
     public Map<String, Object> importProcessDefinition(User loginUser, MultipartFile file, String currentProjectName) {
         Map<String, Object> result = new HashMap<>(5);
         String processMetaJson = FileUtils.file2String(file);
-        List<ProcessMeta> processMetaList = JSON.parseArray(processMetaJson, ProcessMeta.class);
+        List<ProcessMeta> processMetaList = new ArrayList<>();
 
+        try {
+            processMetaList = JSONUtils.getMapper().readValue(processMetaJson, new TypeReference<List<ProcessMeta>>() {});
```

Comment:
why not use JSONUtils.toList

Reply:
toList will return List<LinkedHashMap> and will Cause  exception :
 " Could not write JSON: java.util.LinkedHashMap cannot be cast to org.apache.dolphinscheduler.common.process.Property"